src/components/VmDetails/BaseCard.js

```diff
@@ -118,7 +141,7 @@ BaseCard.propTypes = {
   onStartEdit: PropTypes.func,
   onCancel: PropTypes.func,
   onSave: PropTypes.func,
-  children: PropTypes.func,
+  children: PropTypes.oneOfType([ PropTypes.func, PropTypes.ndoe ]),
```

Comment:
Not `ndoe`, but `node`. Actually I think it can be set as `isRequired`.

Reply:
:man_facepalming:  I'll fix both.